crates/wysiwyg/src/lib.rs

```diff
@@ -12,6 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+extern crate core;
```

Comment:
Shouldn't need this in modern Rust, unless I am confused.

Reply:
I think the IDE automatically added that on some buggy autocompletion 😅 .